plexapi/library.py

```diff
@@ -455,6 +449,49 @@ def all(self, sort=None, **kwargs):
         key = '/library/sections/%s/all%s' % (self.key, sortStr)
         return self.fetchItems(key, **kwargs)
 
+    def hubs(self):
+        """ Returns a list of available `:class:`~plexapi.library.Hub` for this library section.
+        """
+        key = '/hubs/sections/%s' % self.key
+        return self.fetchItems(key)
+
+    def _filters(self):
+        """ Returns a list of :class:`~plexapi.library.Filter` from this library section. """
+        key = '/library/sections/%s/filters' % self.key
+        return self.fetchItems(key, cls=Filter)
+
+    def _sorts(self, mediaType=None):
+        items = []
+        for data in self.listChoices('sorts', mediaType):
+            sort = Sort(server=self._server, data=data._data)
+            sort._initpath = data._initpath
+            items.append(sort)
+        return items
+
+    def filterFields(self, mediaType=None):
+        items = []
+        key = '/library/sections/%s/filters?includeMeta=1' % self.key
+        data = self._server.query(key)
+        for meta in data.iter('Meta'):
+            for metaType in meta.iter('Type'):
+                if mediaType and metaType.attrib.get('type') == mediaType:
```

Comment:
simplify this one.

if not mediatype or ...

Reply:
Yeah, that one was tripping me up. I'm repeating the same for loop for both but I get the mediaType isolated when called. I'll move this PR to draft while I figure it out. I'm open to suggestions, of course.

Reply:
What your doing is fine, but just use the or so you check both conditions at the same time.